packages/Frontend/src/containers/SignInContainer.jsx

```diff
@@ -47,8 +47,15 @@ class SignInContainer extends Component {
     }
 
     const { error, loading } = this.state;
+    let errMessage;
+    if (error && typeof error === 'string') {
+      errMessage = error;
+    }
+    if (error && error.message && typeof error.message === 'string') {
+      errMessage = error.message;
+    }
```

Comment:
I think this can be slightly shortened to this.
`let errMessage;
if (error) {
  errMessage = typeof error === 'string' ? error : error.message;
}`